sklearn/preprocessing/_encoders.py

```diff
@@ -476,6 +496,34 @@ def infrequent_categories_(self):
             for category, indices in zip(self.categories_, infrequent_indices)
         ]
 
+    def _validate_keywords(self):
```

Comment:
@glemaitre My understanding is that we don't need this one anymore since the introduction of parameter constraints. Should I just remove it?

Reply:
Yes, this is taking care by the `_parameter_constraints` indeed.